Makefile

```diff
@@ -1,10 +1,18 @@
 all: lint build unit
 
 OUTPUT_DIR="build/_output"
+# Set the go mod vendor flags, if they're not already set
+GO ?=go
```

Comment:
Why do you need this?

Reply:
We use it [here](https://github.com/openshift/windows-machine-config-operator/pull/74/commits/b6ce7f7919dbc7a9032a57863ce401c5cff8c83b#diff-b67911656ef5d18c4ae36cb6741b7965R6)

Reply:
I understand that it is being used but why do you need it? You should be able to use `go` without it having to be a variable. Or is `$GO` present in CI and you are accounting for it in our local development environment?